Assets/Scripts/UI/CursorMode.cs

```diff
@@ -0,0 +1,5 @@
+﻿public enum CursorMode
+{
+    Menu,
+    Loading
```

Comment:
Should probably change Loading -> Invisible. But in that case, maybe also Menu -> Visible. Thoughts?
@Lithrun

Reply:
I'll keep the implementation separate from the interface. Keeping it as `Menu` and `Loading`.